modules/kubernetes/prometheus/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable "alertmanager_enabled" {
+  description = "If prometheus alertmanager should be enabled or not"
+  type        = bool
+  default     = false
+}
+
+variable "remote_write_enabled" {
+  description = "If remote write should be enabled or not"
+  type        = bool
+  default     = true
+}
+
+variable "remote_write_url" {
+  description = "The URL where to send prometheus remote_write data"
+  type        = string
+}
+
+variable "remote_write_name" {
+  description = "The name of the remote write in prometheus"
+  type        = string
+  default     = "xenitInfra"
+}
+
+variable "remote_tls_secret_name" {
```

Comment:
Where do we configure this secret? Maybe we should include csi secret to be able to read it from KeyVault?

Reply:
Currently we assume that it's there. I started to look in to csi secret to manage it.